src/commcare_cloud/ansible/deploy_commcarehq.yml

```diff
@@ -133,3 +133,10 @@
       with_items: "{{ contents.stdout_lines }}"
       when: item not in supervisor_files|sum(start=[])
   tags: services
+
+- name: Celery tasks cleanup Cron job
+  hosts: celery
+  tasks:
+    - include_tasks: roles/commcarehq/tasks/celery_cron.yml
+  tags: services
+  
```

Comment:
Nitpick: looks like there's trailing whitespace on this line (two spaces?)

Reply:
@dannyroberts sure will make that change.